src/NexusMods.Telemetry/Constants.cs

```diff
@@ -7,10 +7,15 @@ namespace NexusMods.Telemetry;
 [PublicAPI]
 internal static class Constants
 {
-    // NOTE(erri120): don't change this, this has to be in sync with the backend
+    // NOTE(erri120): don't change this, ask me if you have questions
+    public static readonly Uri MatomoTrackingEndpoint = new("https://pw.nexusmods.com/matomo.php");
+    public const string MatomoSiteId = "7";
```

Comment:
I have questions, are the details supposed to be private? Otherwise we should add some more context in the comments, so that somebody can wrangle the meaning if you are not available.

Reply:
There are no details or hidden meaning, this is just the endpoint we're sending data to and the site ID is essentially the namespace for the app, otherwise we'd be mixing app data with website data.

Reply:
We can say that in the comments then. 
`// The site id is essentially the namespace for events coming from the app`